instances/latest/terminologies/deviceType/openMRIScanner.jsonld

```diff
@@ -0,0 +1,18 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/deviceType/openMRIScanner",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/DeviceType",
+  "definition": "An 'open MRI scanner' is a low-field scanner which has a magnetic top and bottom, but is otherwise open, increasing patient's comfort and unobstructed view of the scanning area. The typical magnetic strength lies between 0.2T and 0.3T.",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "open MRI scanner",
```

Comment:
change to "open-bore MRI scanner" make current name a synonym (I prefer to keep consistency for multiple entries with the same superclass)

Reply:
done